camelot/admin/action/field_action.py

```diff
@@ -296,3 +296,74 @@ def model_run(self, model_context):
             destination.write(storage.checkout_stream(stored_file).read())
 
 
+class AddNewObject( FieldAction ):
```

Comment:
Mischien moeten we gewoon 2 AddNewObjects maken, 1 list action voor gebruik op niveau van de 'tabellen', en 1 field action op niveau van de one2many editor.

Dan kunnen ze nog wat gezamelijke functionaliteit hebben via een mixin of zo

Reply:
Er zijn nu 2 AddNewObjects (field & list). Ik moet wel nog de gedeelde functionaliteit in een mixin zetten.